.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,33 +1,30 @@
-<!-- Insert GitHub issue number below -->
-Fixes #
+<!-- Insert GitHub issue number below. An issue is required for all PRs -->
 
-<!-- Select which type of PR this is -->
-- [ ] This PR contains CSS changes
-- [ ] This PR does not contain CSS changes
+Fixes #
```

Comment:
```suggestion
- Fixes #
```

If you do this GitHub has much better styling and actually shows the issue title